package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -300,14 +300,29 @@ bynum *index-range*
     selects atoms 5 through 10 inclusive. All atoms in the
     :class:`MDAnalysis.Universe` are consecutively numbered, and the index
     runs from 1 up to the total number of atoms.
-
+    
+id *index-range*
+    selects all atoms in the given range which consists of two numbers 
+    separated by a colon such as ``id 5:10`` (both inclusive),
+    e.g. ``id 1`` selects all the atoms with id 1; ``id 5:7`` selects all
+    atoms with ids 5, all atoms with ids 6 and all atoms with ids 7.
```

Comment:
Out of curiosity, why did you omit to mention that by default the `id` selection is 1-based? The id as 0-based makes sense only after it is set by the user. Otherwise, the selection will be empty. Example:
```Python3
>>> import MDAnalysis as mda
>>> from MDAnalysis.tests.datafiles import PDB
>>> u = mda.Universe(PDB)
>>> print(u.select_atoms('id 0'))
<AtomGroup []>
>>> print(u.select_atoms('id 1'))
<AtomGroup [<Atom 1: N of type N of resname MET, resid 1 and segid SYSTEM and altLoc >]>
>>> u.atoms.ids = 0
>>> print(u.select_atoms('id 0'))
<AtomGroup [<Atom 1: N of type N of resname MET, resid 1 and segid SYSTEM and altLoc >, ...
```

Reply:
No actually i did mention  in the note that it starts from 1 so i didn't add it here but i made the changes now.